src/main/java/piano/keyboard/keyboardui/Keyboard.java

```diff
@@ -29,8 +29,16 @@ public Keyboard(MidiChannel midiChannel, Recorder recorder) {
 
     private void addWhitePianoLabels() {
         int placement = KeyStats.SPACE_BETWEEN_WHITE_KEYS;
-
         int index = 0;
+
+        addPianoLabel(Color.WHITE, index, placement);
+        placement += KeyStats.WHITE_WIDTH + KeyStats.SPACE_BETWEEN_WHITE_KEYS;
+        index += 2;
+
+        addPianoLabel(Color.WHITE, index, placement);
+        placement += KeyStats.WHITE_WIDTH + KeyStats.SPACE_BETWEEN_WHITE_KEYS;
+        index++;
```

Comment:
Please include a screenshot of the new UI in your PR.

Drag an image to the PR description to upload it.